packages/angular/common/src/providers/menu-controller.ts

```diff
@@ -1,6 +1,7 @@
-import type { MenuControllerI } from '@ionic/core/components';
+import type { MenuControllerI, AnimationBuilder, MenuI } from '@ionic/core/components';
+import { Animation } from '@ionic/core/components';
```

Comment:
Can `Animation` be moved to a type import in the previous import statement?

Reply:
Good catch, thanks!